front/src/common/ScenarioExplorer/ScenarioExplorerTypes.ts

```diff
@@ -1,14 +1,9 @@
-import {
-  projectTypes,
-  scenarioTypes,
-  studyTypes,
-} from 'applications/operationalStudies/components/operationalStudiesTypes';
+import { ProjectResult, ScenarioListResult, StudyResult } from 'common/api/osrdEditoastApi';
 
-export type FilterParams = {
-  name?: string;
-  description?: string;
-  tags?: string;
-  ordering?: string;
+export type ScenarioExplorerProps = {
+  globalProjectId: number | undefined;
```

Comment:
you can said it like this `globalProjectId?: number;`

Reply:
Done